mathesar_ui/src/component-library/icon/Icon.svelte

```diff
@@ -57,6 +60,7 @@
 
   let faClasses: string;
   $: faClasses = concatClasses(classes, flip, rotate);
+  $: dot = hasNotificationDot ? getDot(viewBoxWidth, viewBoxHeight) : undefined;
```

Comment:
I agree adding Dot to Icon is the quickest solution at the moment, so we can go ahead with this but, I'd eventually like to decouple it from the Icon component, and make it more common.

Please open an issue to keep track of this.

Reply:
#2542